src/test/java/org/jenkinsci/plugins/workflow/libs/LibraryAdderTest.java

```diff
@@ -493,7 +493,7 @@ public void parallelBuildsDontInterfereWithExpiredCache() throws Throwable {
                 new SCMSourceRetriever(new GitSCMSource(null, sampleRepo.toString(), "", "*", "", true)));
         config.setDefaultVersion("master");
         config.setImplicit(true);
-        config.setCachingConfiguration(new LibraryCachingConfiguration(30, null));
+        config.setCachingConfiguration(new LibraryCachingConfiguration(30, null,null));
```

Comment:
```suggestion
        config.setCachingConfiguration(new LibraryCachingConfiguration(30, null, null));
```

Reply:
nitpicking

Reply:
Updated in [e0a2432](https://github.com/jenkinsci/pipeline-groovy-lib-plugin/pull/16/commits/e0a243267abfdd351c1610b68985c11cea7eb8be)